index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
```

Comment:
- [ ] To keep your code clean & readable , kindly remove the eslint disabling command and try to fix the errors 🙏